deploy/docker/Dockerfile

```diff
@@ -1,5 +1,10 @@
-FROM BASEIMAGE
+FROM golang:1.13.7 as builder
+COPY . /go/src/github.com/kubernetes-incubator/metrics-server
+RUN cd /go/src/github.com/kubernetes-incubator/metrics-server && \
+    make _output/amd64/metrics-server
 
-COPY metrics-server /
+FROM gcr.io/distroless/static@sha256:08322afd57db6c2fd7a4264bf0edd9913176835585493144ee9ffe0c8b576a76
```

Comment:
Can you bump to the latest nonroot - gcr.io/distroless/static@sha256:23aa732bba4c8618c0d97c26a72a32997363d591807b0d4c31b0bbc8a774bddf

Reply:
Done